code/game/objects/items/devices/flashlight.dm

```diff
@@ -450,6 +452,8 @@
 	item_state = "glowstick"
 	grind_results = list(/datum/reagent/phenol = 15, /datum/reagent/hydrogen = 10, /datum/reagent/oxygen = 5) //Meth-in-a-stick
 	var/fuel = 0
+	toggle_on_sound = 'sound/effects/glowstick.ogg'
+	toggle_off_sound = 'sound/effects/glowstick.ogg'
```

Comment:
how are you going to toggle a glowstick off

Reply:
Eat it 😋